Doc/library/typing.rst

```diff
@@ -3731,9 +3762,6 @@ Aliases to other ABCs in :mod:`collections.abc`
 
    Deprecated alias to :class:`collections.abc.Callable`.
 
-   See :ref:`annotating-callables` for details on how to use
-   :class:`collections.abc.Callable` and ``typing.Callable`` in type annotations.
```

Comment:
Why take this note out? I'd keep it here, and add similar one-sentence links to the new "Annotating generators and coroutines" section from the stub entries for `typing.Generator`, `typing.Coroutine`, etc.

The stub entries for the deprecated objects shouldn't be where the "how to use these type hints" sections are located. But we should make it easy for people who end up at the stub entries to get to the information they wanted to know